src/core/bitop.d

```diff
@@ -119,27 +127,35 @@ unittest
  */
 int bsr(uint v) pure
 {
-    static if (size_t.sizeof == ulong.sizeof)
+    pragma(inline, false);  // so intrinsic detection will work
+    static if (size_t.sizeof == ulong.sizeof)  // 64 bit code gen
```

Comment:
As with `bsf(uint)`, above, you should ditch the `static if` and just always take the `false` branch, which is faster than your revised `true` branch.